Formula/sevenzip.rb

```diff
@@ -0,0 +1,36 @@
+class Sevenzip < Formula
+  desc "7-Zip is a file archiver with a high compression ratio"
+  homepage "https://7-zip.org"
+  url "https://7-zip.org/a/7z2102-src.7z"
```

Comment:
> **7-Zip 21.02 alpha**

Homebrew does not ship alpha versions, we'll have to wait until a stable release.